script/Arcueid/Day 8/08_00_ARC08_2B.txt

```diff
@@ -141,6 +141,7 @@ Rather it was so serious that it was frightening.
 [sha:35ce727002b5ac108b4264549f23a3fc36cbf1a1]{
 -- Page 103, Offset 19926.
 -- なんでもない言葉の中に、こちらの態度を一つも取りこぼさないよう注目する<本|・><気|・>を感じたというか……。
+// i have no idea help please
```

Comment:
wow, this one is rough. Here's sorta what I think:
> なんでもない言葉の中に、こちらの態度を一つも取りこぼさないよう注目する<本|・><気|・>を感じたというか……。
I feel like it's the sorta serious[referencing previous line] where he'll take great care not to miss a single piece of info in anything I say, no matter how trifling.